varcode/effects/effect_classes.py

```diff
@@ -617,14 +617,24 @@ def __init__(
             self,
             variant,
             transcript,
-            extended_protein_sequence):
-        aa_mutation_start_offset = len(transcript.protein_sequence)
+            extended_protein_sequence,
```

Comment:
First time I saw `"*"` in `aa_ref` I was like "what just crashed my script?" (handling a big vcf file, generating mutated protein sequences to a file).

I don't think it's necessary to have the stop codon there, since it's implied that `StopLoss` already affects it.

Being pedantic if you have a variant that affects both sides of the stop codon (e.g. `chr1 100484693 TTCATCTGAACT    CCCTAACCCCCC`), then `aa_ref` would be `FI*T`.